.github/workflows/ci.yaml

```diff
@@ -6,61 +6,52 @@ on:
       - 'jenkins-scripts/dsl/**'
 
 jobs:
-  dsl_ci:
+  xml_generation:
     runs-on: ubuntu-latest
-    name: Diff for DSL code
+    name: Generate XML config from DSL
```

Comment:
According to the upstream docs you need to give the default token scope to change repository contents.
```suggestion
    name: Generate XML config from DSL
    permissions:
      contents: write
```

Reply:
The change looks just correct to me. I was investigating why the action was working on my GitHub fork and found the repository configuration:

![Screenshot from 2023-12-06 16-42-15](https://github.com/gazebo-tooling/release-tools/assets/2098802/4ec91134-6d1e-464d-918c-39b000aa991e)

Might be a good moment to change this.

Reply:
changed.